Code/Framework/AzFramework/AzFramework/Physics/PhysicsScene.h

```diff
@@ -35,6 +35,11 @@ namespace AzPhysics
         //! @returns Will return a SceneHandle to a Scene connected with the given name, otherwise will return InvalidSceneHandle.
         virtual SceneHandle GetSceneHandle(const AZStd::string& sceneName) = 0;
 
+        //! Get the Scene of the requested SceneHandle.
+        //! @param handle The SceneHandle of the requested scene.
+        //! @return Returns a SceneInterface pointer if found, otherwise nullptr.
```

Comment:
The type returning is not `SceneInterface`: `SceneInterface pointer` -> `Scene pointer`

Reply:
nice! the original place I copy-pasted from also has this mistake